src/Core/Services/Implementations/UserService.cs

```diff
@@ -31,6 +31,8 @@
 using File = System.IO.File;
 using JsonSerializer = System.Text.Json.JsonSerializer;
 
+#nullable enable
```

Comment:
Please don't enable nullable unless you've reviewed the code to make sure its null handling is correct. In this case, you've enabled it on this entire class, but if you look at it in the Github diff, there are null warnings everywhere because the existing code doesn't handle nullable types properly.

You should enable it just for your new methods by putting `#nullable enable` immediately before the block of new code, and then `#nullable disable` immediately after. Then double check the Github diff to find and fix any null warnings.

Same for any other files here.